notebooks/HADDOCK3-antibody-antigen.ipynb

```diff
@@ -0,0 +1,2613 @@
+        "* **Scoring modules**\n",
+        "    * `emscoring`: *scoring of a complex performing a short EM (builds the topology and all missing atoms).*\n",
+        "    * `mdscoring`: *scoring of a complex performing a short MD in explicit solvent + EM (builds the topology and all missing atoms).*\n",
+        "* **Analysis modules**\n",
+        "    * `alascan`: *Performs a systematic (or user-define) alanine scanning mutagenesis of interface residues.*\n",
+        "    * `caprieval`: *Calculates CAPRI metrics (i-RMSD, l-RMSD, Fnat, DockQ) with respect to the top-scoring model or reference structure if provided.*\n",
+        "    * `clustfcc`: *Clusters models based on the fraction of common contacts (FCC)*\n",
+        "    * `clustrmsd`: *Clusters models based on pairwise RMSD matrix calculated with the `rmsdmatrix` module.*\n",
+        "    * `contactmap`: *Generate contact matrices of both intra- and intermolecular contacts and a chordchart of intermolecular contacts.*\n",
+        "    * `rmsdmatrix`: *Calculates the pairwise RMSD matrix between all the models generated in the previous step.*\n",
+        "    * `ilrmsdmatrix`: *Calculates the pairwise interface-ligand-RMSD (il-RMSD) matrix between all the models generated in the previous step.*\n",
+        "    * `seletop`: *Selects the top N models from the previous step.*\n",
+        "    * `seletopclusts`: *Selects the top N clusters from the previous step.*\n",
+        "\n",
+        "The HADDOCK3 workflows are defined in simple configuration text files, similar to the TOML format but with extra features.\n",
+        "Contrary to HADDOCK2.X which follows a rigid (yet highly parameterisable)\n",
+        "procedure, in HADDOCK3, you can create your own simulation workflows by\n",
+        "combining a multitude of independent modules that perform specialized tasks.\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "V5b-aupVx27d"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "---\n",
+        "# Software and data setup\n",
+        "\n",
+        "In order to follow this tutorial we will start Jupyter session in Colab or other resources, install haddock3 and download the data for the tutorial.\n",
+        "\n",
+        "To run this tutorial on Colab we recommand changing the runtime type to v2-8 TPU, which will give you access to up to 96 cores. The haddock3 docking run described in this tutorial should complete in about 15 minutes.\n",
+        "\n",
+        "For this follow the following steps:"
+      ],
+      "metadata": {
+        "id": "bs6xG1w1DTaW"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title 1. Install haddock3, BioPython and py3Dmol\n",
+        "#@markdown Execute this cell to download the required packages.\n",
+        "import os\n",
+        "os.chdir('haddock3')\n",
+        "!pip install haddock3[2025.9.3] --quiet\n",
```

Comment:
This is not the correct syntax and this version does not exist - https://pypi.org/project/haddock3/#history

Make sure this version exist and the syntax is correct before merging